scripts/change_url

```diff
@@ -110,17 +112,20 @@ domain="$new_domain"
 path_url="$new_path"
 # port is already defined in this file
 
-ynh_backup_if_checksum_is_different --file="$final_path/config.json5"
```

Comment:
Why remove that ?

Reply:
because it's included in ynh_add_config at https://github.com/YunoHost/yunohost/blob/2b0df6c35aa0273693f108bb887d42c84d1831be/data/helpers.d/utils#L374

Reply:
Good to know, thanks.